workshops/templates/workshops/event.html

```diff
@@ -6,6 +6,17 @@
 {% load tags %}
 
 {% block content %}
+
+{% if event.tags_changed %}
```

Comment:
These things were called tags long before, see `find_tags_on_event_index`, `find_tags_on_event_website`, `parse_tags_from_event_website`, `validate_tags_from_event_website`, `fetch_event_tags` in `util.py`.

I don't know how else you'd these key-value pairs if not tags. I like metatags since they provide structured context (metadata?) about events.

Or maybe metadata would be fine?